### DISCUSSION TITLE
MFA Enforcement

### DISCUSSION THREAD

**Discussion by nirhalfon** [Security-Onion-Solutions/securityonion#12490] [2.4] [answered]
Can you please add MFA enforcement on all users?
Also if possible to add MFA to the other SOS components such as: Elastic, InfluxDB.
Issue is that while a UI (SOC) user logged in with MFA. He can "bypass" the MFA to the other components such as Kibana and using only user@domain \ password.

**Comment by TOoSmOotH** [comment, marked as answer]
Users cannot "bypass" MFA since you must authenticate to SOC before you can access any of these other components.
